apps/dashboard/jest.config.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
```

Comment:
Is there a reason we have a ton of commented configs here?

If we don't need them, let's remove them completely. Thanks